docs/setup/plugins/discourse.md

```diff
@@ -1,5 +1,121 @@
 ---
 title: Discourse
+description: Setting up the Discourse plugin.
 ---
 
-TODO
+## Status and Caveats
+
+This plugin is currently in Beta. It assigns Cred scores that are reasonable and
+robust for a [trust level] 3 community.
+
+Currently there are two general approaches to minting Cred you can take with the
+Discourse plugin. Activity-minted and like-minted Cred (or a hybrid of the two).
+Activity-minted Cred means Cred would be minted for each new topic or post
+created. While like-minted Cred mints new Cred for each like given.
+
+The plugin defaults to using like-minted Cred and for most communities this will
+be the better approach. Unlike activity-minted Cred this allows people to
+validate contributions. People soon learn their likes work as "this is valuable
+and I'd like to see more of this" signals, which incentivizes creating
+high-quality posts.
```

Comment:
For a user who's just encountering the docs, they don't know what any of this means. We could link to some of the relevant discussions on the forums; we also should probably clarify that "setting a node weight on likes but not replies or topics corresponds to like-minted cred, setting a weight on replies or topics corresponds to activity-minted cred"

Reply:
> we also should probably clarify that "setting a node weight on likes but not replies or topics corresponds to like-minted cred, setting a weight on replies or topics corresponds to activity-minted cred"

I feel like I have actually explained this by saying:

> Activity-minted Cred means Cred would be minted for each new topic or post
created. While like-minted Cred mints new Cred for each like given.

Which _only_ assumes that you know what Cred minting is. I feel like we're actually making it more technical if we'd bring in node weights as a proxy for Cred minting.

And it's fair to say people might not know this. But the Discourse setup page isn't the best place to explain that. Linking to an FAQ or Concepts page about minting would be nice :]
I'll track that in an issue. https://github.com/sourcecred/docs/issues/72

As for linking to the forums. I feel like the forums should be a further-reading rabbit hole. If the idea is to make it _more accessible_ for new users, we should port over the key points to the docs, instead of sending them into the rabbit hole.